src/connectedk8s/azext_connectedk8s/_constants.py

```diff
@@ -49,6 +51,40 @@
 Get_Kubernetes_Namespace_Fault_Type = 'kubernetes-get-namespace-error'
 Update_Agent_Success = 'Agents for Connected Cluster {} have been updated successfully'
 Update_Agent_Failure = 'Error while updating agents. Please run \"kubectl get pods -n azure-arc\" to check the pods in case of timeout error. Error: {}'
+User_Not_Found_Type = 'aad-user-not-found-error'
```

Comment:
>User_Not_Found_Type [](start = 0, length = 19)

Where is this used? Are there some unused constants here?

Reply:
Since az proxy is introduced, it might not be used anywhere now.

Reply:
Will remove

Reply:
Removed